src/streaming/streamingclient.py

```diff
@@ -0,0 +1,178 @@
+"""StreamingClient tool used for streaming services"""
+from AudioConfig import AudioConfig
+import websocket
+import io
+import threading
+import six
+import json
+
+def on_error(error):
+    pass
+
+def on_close(code, reason):
+    pass
+
+def on_open():
+    pass
+
+class RevAiStreamingClient():
+    def __init__(self, 
+                access_token, 
+                config, 
+                version = 'v1alpha',  
+                on_error = on_error, 
+                on_close = on_close, 
+                on_open = on_open):
```

Comment:
We should replace this with a function for `on_connected` and call `on_connected` when we receive a connected message from the server. I think this better serves the purpose a user would want, the connected message is a better signifier of successfully beginning a connection with our service AND provides them with their job id